doc/source/selections.rst

```diff
@@ -0,0 +1,344 @@
+.. -*- coding: utf-8 -*-
+.. _selections:
+
+=======================
+Atom selection language
+=======================
+
+AtomGroups can be created by selecting atoms using the MDAnalysis atom selection language:
+
+  >>> kalp = u.select_atoms("segid KALP")
```

Comment:
I am not sure that we have KALP as a data file. Let's use something that we have and can immediately use.

I imagine keen readers to copying and pasting in their own ipython/notebooks to try out what you have.

Reply:
Btw, KALP is a relic of @nmichaud 's original project for which he wrote MDAnalysis.